src/bullets/portfolio/portfolio.py

```diff
@@ -104,3 +105,14 @@ def __put_holding__(self, symbol: str, nb_shares: float, price: float):
             self.holdings.pop(symbol)
         else:
             self.holdings[symbol] = holding
+
+    def __log_market_order__(self, transaction: Transaction):
```

Comment:
This just logs a transaction, could we put this directly in the transaction class?

Reply:
We could change it to log all order types